.github/workflows/certification.yml

```diff
@@ -93,6 +93,8 @@ jobs:
           required-secrets: AzureServiceBusConnectionString
         - component: bindings.azure.cosmosdb
           required-secrets: AzureCosmosDBUrl,AzureCosmosDB,AzureCosmosDBCollection,AzureCosmosDBMasterKey,AzureCertificationTenantId,AzureCertificationServicePrincipalClientId,AzureCertificationServicePrincipalClientSecret
+        - component: bindings.azure.eventhubs
+          required-secrets: AzureEventHubsBindingsConnectionString,AzureBlobStorageAccount,AzureBlobStorageAccessKey,AzureEventHubsBindingsHub,AzureEventHubsBindingsNamespace,AzureCertificationServicePrincipalClientId,AzureCertificationTenantId,AzureCertificationServicePrincipalClientSecret,AzureResourceGroupName,AzureCertificationSubscriptionId,AzureEventHubsBindingsContainer,AzureIotHubEventHubConnectionString,AZURE_CREDENTIALS,iothub-bindings-integration-test,AzureIotHubName,AzureIotHubBindingsConsumerGroup
```

Comment:
Remove AZURE_CREDENTIALS -- that does not belong here

Reply:
Also remove `iothub-bindings-integration-test`

You can only use values found in this list:
https://github.com/dapr/components-contrib/blob/master/.github/infrastructure/conformance/azure/setup-azure-conf-test.sh#L164